src/state/actions/__test__/companies.test.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import * as rtl from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import * as types from '../../types/index';
+
+import { getCompanies, addCompany } from '../companies';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const mock = new MockAdapter(axios);
+
+const testCompanies = [
+  {
+    id: 1,
+    name: 'Accenture',
+    website: 'www.accenture.com.',
+    location: 'Atlanta, GA',
+    type: 'Business',
+    logo: '',
+    description: '',
+    created_at: null,
+    updated_at: null,
```

Comment:
I know this doesn't affect the test but should `created_at` and `updated_at` be null? These are columns that should be automatically created on the companies table using a timestamp like `Date.now()`. Please check the database to verify that these columns are automatically generated.

Reply:
My bad. I grabbed the test data from the readme of the previous repo, should've inspected it. Good spot.